### PR TITLE
Home task 08

### PR DESCRIPTION
**Pull request by isken24**
Home task 08 is ready for review.

### CODE REVIEW

File: hw_08/home_task_08.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+from home_task_06 import create_news, create_private_ad, add_greeting
+import os
+import logging
+import json
+
+
+class JSONReader:
+    def __init__(self, file_path=None):
```
Comment (dena-den) [inline hw_08/home_task_08.py:15]: You can pass alternative path as a default parameter in variable `file_path`. And if-clause will not be needed.

**Review by dena-den — CHANGES_REQUESTED** [review]
You should change format of JSON-file. It must have key-value pairs for each of required parameters.
E.g. 'news_text', 'city' or 'ad_date' keys where it's needed.
Not only 'content' key where you pass everything.

File: hw_08/home_task_08.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+""" Home task 08:
+Expand previous Homework 5/6/7 with additional class, which allow to provide records by JSON file:
+1.Define your input format (one or many records)
+2.Default folder or user provided file path
+3.Remove file if it was successfully processed"""
+
+
+from home_task_06 import create_news, create_private_ad, add_greeting
```
[the remark this answers is not in the document]
Comment (isken24) [reply hw_08/home_task_08.py]: Fixed

File: hw_08/home_task_08.py (hunk 1 above)
Comment (isken24) [reply hw_08/home_task_08.py]: Fixed code as you suggested, but couldn't get rid of if-clause completely, so now it is in `__main__` section :(
Alternative file path should be read from console by using input() function. If user want to use standard file path, he would press Enter, leaving input() empty. And if we try to pass empty `input()` to the `file_path` without additional logic, `file_path` would become equal `""` (empty string) and code would fall with an error.

File: hw_08/home_task_08.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+class JSONParser:
+    def __init__(self, filepath='../hw_08/input_08.json'):
+        self.filepath = filepath
+        self.read_json()
+        self.create_feed_from_json()
+
+    def read_json(self):
+        with open(self.filepath, 'r') as source_file:
+            self.content = json.load(source_file)
+        return self.content
```
Comment (dena-den) [inline hw_08/home_task_08.py:24]: return is excess.

**Review by dena-den — APPROVED** [review]
Get rid of one return and it's approved. Congrats!

**Pull request merged**